examples/android/android-studio/Cukeulator/README.md

```diff
@@ -0,0 +1,72 @@
+## Cukeulator Example Test
+This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
+
+### Setup
+Features must be placed in `assets/features/`. Subdirectories are allowed.
+
+The rest of the dependencies are added automatically in `app/build.gradle`.
+
+The cucumber-android dependency is added as:
+
+```
+androidTestCompile 'info.cukes:cucumber-android:1.2.0@jar'
```

Comment:
@brasmusson apklib does not make any sense to me, since we don't package anything to be used by an app including resources and other android specific things. I will make a pull request to change that.